requests/adapters.py

```diff
@@ -204,11 +198,13 @@ def proxy_manager_for(self, proxy, **proxy_kwargs):
 
         return manager
 
-    def _update_poolmanager_ssl_kw(self, verify, cert):
-        """Update the :class:`PoolManager <urllib3.poolmanager.PoolManager>`
-        connection_pool_kw with the necessary SSL configuration. This method
-        should not be called from user code, and is only exposed for use when
-        subclassing the :class:`HTTPAdapter <requests.adapters.HTTPAdapter>`.
+    @staticmethod
+    def _pool_kwargs(verify, cert):
```

Comment:
I can't think of a particularly good reason for someone to want to override this, so a function sounds fine me. If someone really objects to the behaviour for some unforeseen reason, they can override ``get_connection`` and build the pool kwargs their own way.